meetings/Agenda_2017-09-03.md

```diff
@@ -0,0 +1,24 @@
+## Meeting Agenda, September 3, 2017
+
+1. **Fall events (5 min)**
+ * New meeting time: Thursday 6 pm (most popular slot)
+ * Possibly add Tuesday 4-6 as a coworking session? That way everyone who filled out the doodle can come. 
```

Comment:
Don't forget that we have office hours for the RQM course Tuesdays 4-5.

Reply:
Right. We can always make sure that one of us is at either of the coworking or office hours

Reply:
Did we want to do this every week or is it less frequent?

Reply:
Also I can't remember if we wanted to pick a new place for these meetings?

Reply:
@linamnt - maybe every other week? I think it would be nice to do it in a cafe or something for a nice atmosphere. Also maybe a different side of campus to mix things up. Any suggestions?

Reply:
I like Hart House, it's larger so we're likely to be able to find seats. Also the University College Commuter Lounge/Diabolo's Cafe is pretty large with big tables to work at.